pkg/bpf/bpf_linux.go

```diff
@@ -162,7 +162,7 @@ func LookupElementFromPointers(fd int, structPtr unsafe.Pointer, sizeOfStruct ui
 	}
 
 	if ret != 0 || err != 0 {
-		return fmt.Errorf("Unable to lookup element in map with file descriptor %d: %s", fd, err)
```

Comment:
`%w` is fairly recent addition, does version of Go use in 1.7 support it?

Reply:
Looks like 1.7 is on Go 1.13, so luckily it was the first version where [this feature was added](https://blog.golang.org/go1.13-errors)!